doc/rds-day-2.md

```diff
@@ -0,0 +1,98 @@
+# RDS day 2 operations 
+
+
+## Background and problem statement
+
+
+Managing databases can be challenging because they are stateful, not easily replaceable, and data loss could have significant business impact. An unexpected restart could cause havoc to applications that depend on them. Because of this, people want to offload management, maintenance, and availability of databases to another entity such as cloud providers. Amazon RDS is one of such services. 
+Crossplane AWS provider aims to create building blocks for self-service experience for developers by providing abilities to manage AWS resources in Kubernetes native ways. 
+
+In Amazon RDS some operations require an instance restart. For example, version upgrade and storage size modification require an instance restart. RDS attempts to minimize impact of such operations by:
+1. Define a scheduled maintenance window.
+2. Queue changes that you want to make.
+3. During the next scheduled maintenance window, changes are applied.
+
+This approach is fundamentally different from GitOps. In GitOps, when a change is checked into your repository, it is expected that actual resources are to match the specifications provided in the repository. 
+
+RDS supports applying these changes immediately instead of waiting for a scheduled maintenance window, and when using Crossplane AWS providers, they have the option to apply changes immediately as well. This is the option that should be used when using RDS with GitOps. However this leads to problems when enabling self service model where developers can provision resources on their own. 
+
```

Comment:
There is the config changes inside the db engine via parameter_groups, that will require a reboot, or user wants to queue the config change for a reboot. 
Maybe we want to include this use case on how to handle paramter_groups, maybe is validating that a ticket is need it to change certain parameters, or just be part of normal notification that any change to the DB managed resource in prod a notification is sent out or a ticket is open to record the change.